docs/strings.md

```diff
@@ -0,0 +1,397 @@
+But code using `Temporal.PlainDate.from` will encounter off-by-one-day bugs, which can be maddening to track down because they're intermittent (only show up in some time zones and only at some times of day) and aren't attributable to code changes in the client app.
+
+To prevent these types of bugs, an exception will be thrown if a `Temporal.PlainXxx` type is asked to parse a timestamp string with a `Z` suffix.
+
+#### In the unusual case where I want to get the calendar/clock units from a UTC "Z" string, how can I do that?
+
+In rare cases, developers may want to extract the date and time units from a timestamp string that uses a `Z` suffix.
+The answer above explains why this is usually a bad idea, but sometimes it's needed.
+For example, code that stores a new log file every day may need to flip over to a new file at midnight UTC.
+To handle cases like this, first parse the string into a `Temporal.Instant` and then convert to a `Temporal.ZonedDateTime` using the `UTC` time zone.
+
+```javascript
+s = `2022-02-01T00:00Z`;
+Temporal.PlainDate.from(s); // throws
+zdt = Temporal.Instant.from(s).toZonedDateTimeISO(s).toPlainDate(); // => 2022-02-01
+```
+
+#### Can I parse a subset of data in a string?
+
+Yes!
+Temporal parsing methods accept strings that have more information than the type needs.
+Information not needed for that type's data model is ignored.
+This works for all `Temporal.PlainXxx` types, `Temporal.Instant`, and even `Temporal.TimeZone` and `Temporal.Calendar`.
+
+```javascript
+s = '2022-02-28T11:06:00.092121729+08:00[Asia/Shanghai]';
+Temporal.ZonedDateTime.from(s); // => 2022-02-28T11:06:00.092121729+08:00[Asia/Shanghai]
+Temporal.PlainTime.from(s); // => 11:06:00.092121729
+Temporal.PlainDate.from(s); // => 2022-02-28
+Temporal.PlainDateTime.from(s); // => 2022-02-28T11:06:00.092121729
+Temporal.PlainYearMonth.from(s); // => 2022-02
+Temporal.PlainMonthDay.from(s); // => 02-28 (note: may change to "XXXX-02-28")
+Temporal.Instant.from(s); // => 2022-02-28T03:06:00.092121729Z
+Temporal.TimeZone.from(s); // => Asia/Shanghai
+```
+
+<a name="localized-parsing"></a>
+
+#### Why doesn't `Temporal` support parsing localized formats like MM/DD/YY?
+
+Parsing of localized date/time formats is notoriously difficult and brittle.
+For example, here's what MDN has to say about [`Date.parse`]()
+
+> It is not recommended to use `Date.parse` as until ES5, parsing of strings was entirely implementation dependent.
+> There are still many differences in how different hosts parse date strings, therefore date strings should be manually parsed (a library can help if many different formats are to be accommodated).
+
+Because of the complexity involved in standardizing and implementing localized date parsing, TC39 (the standards committee responsible for ECMAScript) has learned from the painful experience of `Date.parse` that localized parsing is best left to userland libraries, not platform specifications.
```

Comment:
I usually point people to the discussion in https://github.com/tc39/ecma402/issues/342#issuecomment-486857111 for a discussion of strict vs. lenient localized parsing, and in-depth reasons why ECMA-402 wouldn't implement a lenient localized parser (and therefore it would be a non-starter for Temporal.)

We could also point people to https://github.com/js-temporal/proposal-temporal-v2/issues/2 for this, which deals with strict parsing rather than lenient.